assets/constants.js

```diff
@@ -609,5 +609,11 @@ var constants = {
   // eslint-disable-next-line max-len
   "DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR": "Check out this interactive lesson I created on Oppia - a free platform for teaching and learning!",
 
+  // eslint-disable-next-line max-len
```

Comment:
Is this really needed? Can't you just split the key and value into 2 separate lines?

Reply:
Splitted